src/test/resources/net/masterthought/cucumber/location_bug.json

```diff
@@ -0,0 +1,114 @@
+[
+  {
+    "uri": "features/selenium.feature",
+    "id": "test",
+    "keyword": "Feature",
+    "name": "test",
+    "description": "",
+    "line": 1,
+    "elements": [
+      {
+        "id": "test;test1",
+        "keyword": "Scenario",
+        "name": "test1",
+        "description": "",
+        "line": 2,
+        "type": "scenario",
+        "steps": [
+          {
+            "keyword": "Given ",
+            "name": "We are using selenium to test with chrome",
+            "line": 3,
+            "match": {
+              "location": "features/step_definitions/selenium/selenium.rb:8"
+            },
+            "result": {
+              "status": "passed",
+              "duration": 1717150000
+            }
+          },
+          {
+            "keyword": "When ",
+            "name": "I navigate to \"http://www.google.com\"",
+            "line": 4,
+            "match": {
+              "location": "selenium-cucumber-2.1.3/lib/selenium-cucumber/navigation_steps.rb:3"
+            },
+            "result": {
+              "status": "passed",
+              "duration": 1371137000
+            }
+          },
+          {
+            "keyword": "Then ",
+            "name": "I should see page title as \"Google\"",
+            "line": 5,
+            "match": {
+              "location": "selenium-cucumber-2.1.3/lib/selenium-cucumber/assertion_steps.rb:4"
+            },
+            "result": {
+              "status": "passed",
+              "duration": 11001000
+            }
+          },
+          {
+            "keyword": "And ",
+            "name": "element having id \"lst-ib\" should be enabled",
+            "line": 6,
+            "match": {
+              "location": "selenium-cucumber-2.1.3/lib/selenium-cucumber/assertion_steps.rb:31"
+            },
+            "result": {
+              "status": "passed",
+              "duration": 33003000
+            }
+          },
+          {
+            "keyword": "When ",
+            "name": "I enter \"hello world\" into input field having id \"lst-ib\"",
+            "line": 7,
+            "match": {
+              "location": "selenium-cucumber-2.1.3/lib/selenium-cucumber/input_steps.rb:4"
+            },
+            "result": {
+              "status": "passed",
+              "duration": 82008000
+            }
+          },
+          {
+            "keyword": "And ",
+            "name": "I click on element having id \"lst-ib\"",
+            "line": 8,
+            "match": {
+              "location": "selenium-cucumber-2.1.3/lib/selenium-cucumber/click_elements_steps.rb:4"
+            },
+            "result": {
+              "status": "passed",
+              "duration": 43005000
+            }
+          }
+        ],
+        "after": [
+          {
+            "match": {
+              "location": {
+                "filepath": {
```

Comment:
I expect that different format may be also available in scenario, not only hook. Maybe it's worth to add extended format to scenario and validate it on html page? What do you think?